packages/inscription-protocol/src/data/inscription.ts

```diff
@@ -515,7 +520,6 @@ export interface StartPermission {
 export interface ElementWebServiceCall {
   output: ScriptMapCode;
   cache: Cache;
-  clientId: string;
```

Comment:
@ivy-rew is this now intended?

Reply:
nope; fix is on the way https://github.com/axonivy/core/pull/8265

Reply:
I'll revert this thing here.